a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -250,7 +250,7 @@ def get_new_lb_networking_subflow(self, topology):
         new_LB_net_subflow.add(a10_network_tasks.ApplyQos(
             requires=(constants.LOADBALANCER, constants.AMPS_DATA,
                       constants.UPDATE_DICT)))
-        new_LB_net_subflow.add(database_tasks.UpdateAmphoraVIPData(
+        new_LB_net_subflow.add(database_tasks.UpdateAmphoraeVIPData(
```

Comment:
Yikes that's not good to see this now. I missed this change in PR #33 . Please make sure to functionally test PRs before submitting them. This would have been caught sooner.

Reply:
The machine to test PR #33 was compatible with Rocky. Hence stein changes got overridden, fixed those changes in this PR.